content/v2/webhooks/events.markdown

```diff
@@ -90,6 +90,7 @@ The following events are available:
 * domain.auto\_renewal\_enable
 * domain.create
 * domain.delete
+* domain.lock
 * domain.register:started
```

Comment:
Not related to this PR, but when changes were made in https://github.com/dnsimple/dnsimple-developer/pull/505 it seems we did not keep names sorted properly. Can you please have state-aware events (the ones ending with `:`) added at the end?

```
* domain.transfer
* domain.transfer:started
* domain.transfer:cancelled
```

vs

```
* domain.transfer:started
* domain.transfer
* domain.transfer:cancelled
```

This for all cases in this file.

Reply:
These have all been alphabetized with state-aware events at the end of each section.

```
* domain.transfer
* domain.transfer:cancelled
* domain.transfer:started
```